In [3]:
import pandas as pd

**Задача 40:**
*Работать с файлом california_housing_train.csv, 
который находится в папке sample_data.*
* Определить среднюю стоимость дома, 
где кол-во людей от 0 до 500 (population).

In [4]:
df = pd.read_csv('california_housing_train.csv') 

In [5]:
df[df['population'] <= 500][['median_house_value']].mean()

median_house_value    206799.951402
dtype: float64

Ответ: median_house_value    206799.951402

**Задача 42:**
* Узнать какая максимальная households в зоне минимального значения population.

In [6]:
df[df['population'] == df['population'].min()]['households'].max()

4.0

Ответ: 4.0